web-api/runtimes/puppeteer/package.json

```diff
@@ -10,8 +10,8 @@
   "dependencies": {
     "@sparticuz/chromium": "123.0.1",
     "pug": "3.0.2",
-    "puppeteer-core": "22.6.5",
-    "sass": "1.75.0"
+    "puppeteer-core": "22.7.1",
```

Comment:
[Puppeteer ships with a preferred version of chromium](https://pptr.dev/supported-browsers), it looks like Puppeteer 22.7.1 supports chromium 124.0.6. Just wanted to point that out and make sure that PDF generation is working 😄

Reply:
It is and we'll be sure to update chromium with the next update this week.